.eslintrc.yml

```diff
@@ -37,8 +38,8 @@ overrides:
     globals:
       expect: no
   - files:
-    - bin/*
-    - lib/**/*.js
+      - bin/*
+      - lib/**/*.js
```

Comment:
Remove extra indentation.

Reply:
This is from auto formatting and is in line with the corresponding blocks above and below